src/RIGID/fix_rigid_small.cpp

```diff
@@ -1110,11 +1110,6 @@ void FixRigidSmall::enforce2d()
     b->angmom[1] = 0.0;
     b->omega[0] = 0.0;
     b->omega[1] = 0.0;
-    if (langflag && langextra) {
```

Comment:
I think we still need this part because FixRigidSmall doesn't support tflag and fflag (whereas FixRigid does). In the example in.micelle-rigid, the fix rigid/small command does not have the langevin option.